app/views/layouts/application.html.erb

```diff
@@ -72,21 +52,31 @@
           <%= yield %>
         </div>
         <!-- End Content -->
-
-      </div>
-      <!-- End Page Body -->
-      <!-- Footer -->
-    </div>
-    <div id="footer">
+        <div id="footer">
         <a href="http://autolabproject.com">Autolab Project</a>
         · <a href="/contactUs.html">Contact</a>
         <% if @course %>
         · <%= link_to "Need Help?", course_report_bug_path(@course) %>
         <% end %>
-        · <%= link_to "Logout", destroy_user_session_path, method: :delete %>
+        </div>
+        <!-- End Footer -->
+      </div>
+      <!-- End Page Body -->
+      <!-- Footer -->
     </div>
-    <!-- End Footer -->
-    <%= yield :javascript %>
 
+    <%= external_javascript_include_tag "jquery", "2.1.1" %>
+    <%= external_javascript_include_tag "bootstrap", "3.0.0" %>
+    <%= javascript_include_tag 'application' %>
+    <%= yield :javascript %>
+    <script type="text/javascript">
+      /*Scripts to load when the page is ready*/
+      // this does work
```

Comment:
Maybe have a (Ruby?) comment explaining that this allows jquery ajax requests to be authenticated.  This bit of code is actually only used on the gradesheet and perhaps in a few other places, as far as i know